src/functionaltest/java/com/ericsson/ei/frontend/TestAlarm.java

```diff
@@ -5,39 +5,53 @@
 import java.io.IOException;
 
 import org.apache.http.impl.client.CloseableHttpClient;
+import org.junit.Before;
 import org.junit.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.JavascriptExecutor;
 import org.springframework.boot.test.mock.mockito.MockBean;
 
-import com.ericsson.ei.frontend.pageobjects.IndexPage;
 import com.ericsson.ei.frontend.pageobjects.TestRulesPage;
 
 public class TestAlarm extends SeleniumBaseClass {
 
     @MockBean
     protected CloseableHttpClient mockedHttpClient;
 
-    @Test
-    public void testAlarm() throws IOException {
+    private JavascriptExecutor js;
```

Comment:
js is short for Javascript, I would like to see a better name for this variable.

Reply:
Removed it since the driver could execute scripts as well.